test/test_browser.py

```diff
@@ -2646,6 +2646,9 @@ def test_html5_gamepad(self):
       print(opts)
       self.btest_exit(test_file('test_gamepad.c'), args=[] + opts)
 
+  def test_html5_unknown_event_target(self):
+    self.btest_exit(test_file('html5_unknown_event_target.cpp'))
```

Comment:
Can you name the file that same as the test itself (e.g. `test_html5_unknown_event_target.cpp`)